src/components/edit-and-share/edit-audio/changeAudio/index.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Button } from 'antd';
+import { CloseOutlined, PaperClipOutlined } from '@ant-design/icons';
+import UploadAudio from './UploadAudio';
+import 'antd/dist/antd.css';
+
+class ChangeAudio extends React.Component {
+  state = {
+    change: false
+  };
+  handleChange = () => this.setState({ change: true })
+  render() {
+    return (
```

Comment:
Consistencia visual entre los archivos:
```suggestion
  state = {
    change: false
  };

  handleChange = () => this.setState({ change: true })

  render() {
    return (
```